### PR TITLE
Add IBM Cloud MFA

### PR DESCRIPTION
**Pull request by dasilvatek**
Cloud: replaced legacy IBM Bluemix with IBM Cloud platform and added MFA
~Developer: added IBM Cloud App ID for developing MFA within applications~

### CODE REVIEW

File: _data/developer.yml [hunk 1]
```diff
@@ -266,6 +266,14 @@ websites:
       sms: Yes
       doc: https://www.vagrantup.com/docs/vagrant-cloud/users/authentication.html#two-factor-authentication
 
+    - name: IBM Cloud App ID
```
Comment (Carlgo11) [inline _data/developer.yml]: IBM Cloud App ID is what I'd classify as a authentication provider. We no longer list [providers](https://twofactorauth.org/providers/).
Please remove IBM Cloud App ID from this pull request.

**Review by Carlgo11 — CHANGES_REQUESTED** [review]

File: _data/developer.yml (hunk 1 above)
Comment (dasilvatek) [reply _data/developer.yml]: Done and my apologies.

**Comment by Carlgo11** [comment]
I don't see anything about email- or sms-based authentication. Could you link me to any documentation about that?

**Comment by dasilvatek** [comment]
You're right about Email and I will remove.
Phone based is in the last paragraph of the docs: https://cloud.ibm.com/docs/iam?topic=iam-types
Additional detail here: https://cloud.ibm.com/docs/customer-portal?topic=customer-portal-cp_setup-2fa

**Comment by Carlgo11** [comment]
ah okay so you can add `phone` to the list as well :wink:

**Comment by dasilvatek** [comment]
@Carlgo11 done ~- but now it's failing checks.. any idea why?~

**Review by dsoegijono — APPROVED** [review]

**Review by Carlgo11 — APPROVED** [review]

**Pull request merged**